src/vs/workbench/parts/tasks/electron-browser/media/task.contribution.css

```diff
@@ -69,4 +69,78 @@
 .vs-dark .monaco-workbench .quick-open-task-configure,
 .hc-black .monaco-workbench .quick-open-task-configure {
 	background-image: url('configure-inverse.svg');
-}
\ No newline at end of file
+}
+
+
+/*
+ * Task Panel styles
+ */
+
+.monaco-workbench .task-action { background: url('update.svg') center center no-repeat; }
+
+.no-telemetry {
+	background-image: url('thumbs-down.svg');
+	width: 50px;
+	height: 50px;
+	vertical-align: middle;
+}
+
+.yes-telemetry {
+	background-image: url('thumbs-up.svg');
+	width: 50px;
+	height: 50px;
+	vertical-align: middle;
+}
+
+.task-panel {
+	padding-left: 10px;
+	padding-right: 10px;
+	margin-right: 10%;
+	margin-left: 10%;
+}
+
+.task-panel-container {
+	outline: grey dotted thin;
+	padding: 1px 17px;
+	margin-top: 10px;
+}
+
+.task-item {
+	outline: grey solid thin;
+	padding-left: 5px;
+	padding-right: 5px;
+	padding: 1px;
+	margin-bottom: 10px;
+}
+
+.mockup-button {
+	display: inline-block;
+	padding: 2px;
+}
+
+.oneliner {
+	text-align: left;
+	margin: 10px;
+}
+
+.right-aligned {
+	float: right;
+}
+
+.centered {
+	text-align: center;
+	margin-top: 10px;
+}
+
+.header-item {
+	font-size: large;
```

Comment:
Use a `px` size for font-sizes, I don't know if `large`, `smaller` etc are consistent.